osf/models/contributor.py

```diff
@@ -32,11 +32,6 @@ def permission(self):
 
 
 class Contributor(AbstractBaseContributor):
-    # TO BE DELETED (read/write/admin fields)
-    read = models.BooleanField(default=False)
-    write = models.BooleanField(default=False)
-    admin = models.BooleanField(default=False)
```

Comment:
quick drive by comment, InstitutionalContributor read/write/admin columns need to be removed as well!  I should have specified this -

Reply:
Removed!